tests/test_xcode_dependencies.lua

```diff
@@ -12,26 +12,36 @@
 -- Setup/Teardown
 ---------------------------------------------------------------------------
 
-	local wks, prj, prj2, tr
+	local wks, prj1, prj2, prj3, tr
 
 	function suite.teardown()
 		wks = nil
-		prj = nil
+		prj1 = nil
 		prj2 = nil
+		prj3 = nil
 		tr = nil
 	end
 
 	function suite.setup()
 		_ACTION = "xcode4"
 		xcode.used_ids = { } -- reset the list of generated IDs
 
-		wks, prj = test.createWorkspace()
+		wks, prj1 = test.createWorkspace()
```

Comment:
hmmm, I dont know =) I was trying to use the same logic/format as the other tests that were already in place. Certainly dont need multiple workspaces.